.ideainspect

```diff
@@ -1,12 +1,6 @@
 # Levels to look for. Default: WARNING,ERROR
 levels: WARNING,ERROR,INFO
 
-# Inspection result files to skip. For example "TodoComment" or "TodoComment.xml".
```

Comment:
I think the option to ignore inspections based on issue type is and will be always useful. 

For example you might want to receive i.e. hints & warnings i.e. about code style but do not intend to break you CI one those.